File: BackEnd_Learn/Java/jwt/src/main/java/com/example/jwt/Entity/RefreshToken/RefreshTokenRepository.java
package com.example.jwt.Entity.RefreshToken;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.Optional;

@Repository
public interface RefreshTokenRepository extends JpaRepository<RefreshToken, Integer> {
    Optional<RefreshToken> findById(Integer id);
    Optional<RefreshToken> findByRefreshToken(String RefreshToken);
}

File: BackEnd_Learn/kotlin/kopring/src/main/kotlin/io/github/leeseojune53/kopring/domain/refresh_token/domain/repositories/RefreshTokenRepository.kt
package io.github.leeseojune53.kopring.domain.refresh_token.domain.repositories

import io.github.leeseojune53.kopring.domain.refresh_token.domain.RefreshToken
import org.springframework.data.repository.CrudRepository

interface RefreshTokenRepository: CrudRepository<RefreshToken, String> {
}